public/app/features/alerting/unified/api/alertingFolderActionsApi.ts

```diff
@@ -8,7 +8,10 @@ export const alertingFolderActionsApi = alertingApi.injectEndpoints({
   endpoints: (build) => ({
     pauseFolder: build.mutation<void, WithNotificationOptions<{ namespace: string }>>({
       query: ({ namespace, notificationOptions }) => {
-        const successMessage = t('alerting.bulk-actions.pause.success', 'Rules successfully paused for this folder');
+        const successMessage = t(
+          'alerting.bulk-actions.pause.success',
+          'Rules evaluation successfully paused for folder'
+        );
```

Comment:
A few text changes throughout but we can discuss on the experience call tomorrow